main/migrations/0010_auto_20151118_0825.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.db.models.deletion
+import django.core.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0009_auto_20151116_1022'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
For example this line is super suspicious, as the `building` model already exists...